test/cpp/jit/tests.h

```diff
@@ -849,7 +849,7 @@ const auto cf_examples = R"JIT(
 void testControlFlow() {
   script::Module cu;
   script::defineMethodsInModule(
-      cu, cf_examples, torch::jit::script::nativeResolver, nullptr);
+      cu, cf_examples, std::shared_ptr<torch::jit::script::NativeResolver>(), nullptr);
```

Comment:
Why not pass a `nullptr` if that's what you wanted to do (unless you really wanted `make_shared`)?

Reply:
Because this was a regex find-replace :D I'll fix it